DataDefinitions/EddiDataDefinitions.csproj

```diff
@@ -663,6 +668,10 @@
       <Generator>PublicResXFileCodeGenerator</Generator>
       <LastGenOutput>AsteroidMaterialContent.Designer.cs</LastGenOutput>
     </EmbeddedResource>
+    <EmbeddedResource Include="Properties\EngineerSpecialty.resx">
```

Comment:
Don't we need to add the translation resources too?

Reply:
I was under the impression that those would be created automatically with CrowdIn, but you know better than I do. Easy to add in any case.

Reply:
I mean, we need to add them to the csproj so that they get compiled. CrowdIn doesn't touch csproj files (thank God).